### PR TITLE
Updates shoukou up to date.

### PR DESCRIPTION
**Pull request by Adrian16199**
Again

This time at the least, I noticed that the lights were changed so at the least the map was updated properly before i touched it.

Changes cloning to metem.
Redo captain's quarters to include all his gear.
Added a noticeboard to kitchen.

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

File: Resources/Maps/shoukou.yml [hunk 1]
```diff
@@ -28727,6 +28720,14 @@ entities:
       rot: 1.5707963267948966 rad
       pos: 6.5,-41.5
       parent: 34
+    - type: DeviceLinkSource
```
Comment (IamVelcroboy) [inline Resources/Maps/shoukou.yml:28723]: These look linked wrong. When you're linking, link the computer to the scanner MedicalScannerSender->MedicalScannerSender. Then link the computer to the pod CloningPodSender->CloningPodSender.

File: Resources/Maps/shoukou.yml [hunk 2]
```diff
@@ -57805,12 +57801,12 @@ entities:
     - type: Transform
       pos: 19.5,1.5
       parent: 34
-- proto: LockerCaptainFilledHardsuit
```
Comment (IamVelcroboy) [inline Resources/Maps/shoukou.yml:57808]: Does the captain still have a hardsuit?

File: Resources/Maps/shoukou.yml [hunk 3]
```diff
@@ -69029,6 +69031,13 @@ entities:
     - type: Transform
       pos: -47.5,-29.5
       parent: 34
+- proto: SuitStorageCaptain
```
Comment (IamVelcroboy) [inline Resources/Maps/shoukou.yml:69034]: nm, here it is

File: Resources/Maps/shoukou.yml [hunk 4]
```diff
@@ -28727,6 +28720,14 @@ entities:
...
       pos: 6.5,-41.5
       parent: 34
+    - type: DeviceLinkSource
+      linkedPorts:
+        4263:
+        - MedicalScannerSender: CloningPodReceiver
+        - CloningPodSender: CloningPodReceiver
+        12439:
+        - MedicalScannerSender: MedicalScannerReceiver
+        - CloningPodSender: MedicalScannerReceiver
```
Comment (IamVelcroboy) [inline Resources/Maps/shoukou.yml:28730]: These look linked wrong. When you're linking, link the computer to the scanner MedicalScannerSender->MedicalScannerSender. Then link the computer to the pod CloningPodSender->CloningPodSender.

(moved down for better clarity)

Comment (Adrian16199) [reply Resources/Maps/shoukou.yml:28730]: I mean, I link the computers exactly how I know to make sure that they actualy work.

**Review by IamVelcroboy — APPROVED** [review]

**Pull request merged**